Add source links to the open-telemetry documentation (#528)
digital-asset/docs.daml.com: docs/2.8.0/docs/app-dev/bindings-java/open-tracing.rst

To use the OpenTelemetry libraries, add the following **Maven** dependencies to your project's ``pom.xml``:

.. remoteliteralinclude:: https://raw.githubusercontent.com/digital-asset/ex-java-bindings-with-opentelemetry/master/pom.xml
   :language: xml
   :lines: 33-52

.. note::

>     Replace the version number in all dependencies with the version that you want to use. You can find the available versions by checking the `Maven Central Repository <https://search.maven.org/artifact/io.opentelemetry/opentelemetry-api>`__ etc.

Reviewer: ```suggestion
    Replace the version number in each dependency with the version you want to use. To find available versions, check the `Maven Central Repository <https://search.maven.org/artifact/io.opentelemetry/opentelemetry-api>`__.
```

Reviewer: The Maven search doesn't show anything for me... maybe double-check the link

Author: It gets me to this page:
![image](https://github.com/digital-asset/docs.daml.com/assets/28315767/2f9af8a4-d181-4565-b56c-317a51780d42)

Author: I have also asked a few colleagues to check, and it works for them too, :thinking:

Author: It also works for me when I click it from the current documentation page (little note at the bottom):
https://docs.daml.com/2.8.0/app-dev/bindings-java/open-tracing.html#add-project-dependencies

Reviewer: Thanks, the link works fine for me now. I must have hit a Maven hiccup or outage yesterday.
